droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/CsvItemWriter.java

```diff
@@ -162,6 +190,9 @@ public void write(List<? extends ProfileResourceNode> nodes) {
     }
     
     private void writeOneRowPerFile(List<? extends ProfileResourceNode> nodes) {
+        if (csvWriter.getRecordCount() == 0) {
+            writeHeadersForOneLinePerFileExport(nodes);
```

Comment:
Sorry for being pedantic, could this be `row` instead of `line` as everything else in this file refers to lines as rows?